tests/unit/utils/app-icon-extractor.test.js

```
@@ -65,11 +66,11 @@ function setupMockScenario(scenario) {
         return Buffer.from([0x69, 0x63, 0x6e, 0x73]); // icns header
       });
       mockFs.statSync.mockReturnValue({ size: 2048 });
-      mockChildProcess.execSync
+      mockChildProcess.execFileSync
         .mockImplementationOnce(() => { throw new Error('iconutil failed'); })
```

ESLint

[error] Statement inside of curly braces should be on next line.

(brace-style)